central/policy/service/customresource/policy.go

```diff
@@ -0,0 +1,171 @@
+// TODO(ROX-26318): Codes to be generated later.
+// Code generated by policy-as-code-helper generator. DO NOT EDIT.
+package customresource
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sliceutils"
+)
+
+// Scope represents storage.Scope in the Custom Resource.
+type Scope struct {
+	Cluster   string               `yaml:",omitempty"`
+	Namespace string               `yaml:",omitempty"`
+	Label     *storage.Scope_Label `yaml:",omitempty"`
+}
+
+// convertScope Converts storage.Scope to *Scope
+func convertScope(p *storage.Scope) *Scope {
+	if p == nil {
+		return nil
+	}
+
+	return &Scope{
+		Cluster:   p.Cluster,
+		Namespace: p.Namespace,
+		Label:     p.Label,
+	}
+}
+
+// Exclusion_Deployment represents storage.Exclusion_Deployment in the Custom Resource.
+type Exclusion_Deployment struct {
+	Name  string `yaml:",omitempty"`
+	Scope *Scope `yaml:",omitempty"`
+}
+
+// convertExclusion_Deployment Converts storage.Exclusion_Deployment to *Exclusion_Deployment
+func convertExclusion_Deployment(p *storage.Exclusion_Deployment) *Exclusion_Deployment {
+	if p == nil {
+		return nil
+	}
+
+	return &Exclusion_Deployment{
+		Name:  p.Name,
+		Scope: convertScope(p.Scope),
+	}
+}
+
+// Exclusion represents storage.Exclusion in the Custom Resource.
+type Exclusion struct {
+	Name       string                   `yaml:",omitempty"`
+	Deployment *Exclusion_Deployment    `yaml:",omitempty"`
+	Image      *storage.Exclusion_Image `yaml:",omitempty"`
+	Expiration string                   `yaml:",omitempty"`
```

Comment:
do none of these fields need the name of the yaml field in the tag?

Reply:
omitempty is not by default.